downstream/modules/eda/proc-eda-set-up-credential-types.adoc

```diff
@@ -0,0 +1,169 @@
+[id="eda-set-up-new-credential-types"]
+
+= Creating a new credential type
+
+You can create a credential type to use with a source plugin or a private container registry that you select based on the supported, default credential types. You can make your credential type available to a team or individuals.
+
+
+.Procedure
+
+. Log in to the {PlatformNameShort} Dashboard.
+. From the navigation panel, select {MenuADCredentialType}.
+. Click btn:[Create credential type].
+. Insert the following:
++
+Name:: Insert the name.
+Description:: This field is optional.
+. In the *Input Configuration* field, specify an input schema that defines a set of ordered fields for that type. The format can be in YAML or JSON:
++
+*YAML*
++
+[literal, options="nowrap" subs="+attributes"]
+----
+fields:
+  - type: string
+    id: username
+    label: Username
+  - type: string
+    id: password
+    label: Password
+    secret: true
+required:
+  - username
+  - password
+----
++
+
+View more YAML examples at the link:https://yaml.org/spec/1.2.2/[YAML page].
++
+*JSON*
++
+[literal, options="nowrap" subs="+attributes"]
+----
+{
+"fields": [
+  {
+  "type": "string",
+  "id": "username",
+  "label": "Username"
+  },
+  {
+  "secret": true,
+  "type": "string",
+  "id": "password",
+  "label": "Password"
+   }
+  ],
+ "required": ["username", "password"]
+}
+----
++
+View more JSON examples at link:https://www.json.org/json-en.html[The JSON website].
+
+
+
+. In the *Injector Configuration* field, enter environment variables or extra variables that specify the values a credential type can inject.
+The format can be in YAML or JSON (see examples in the previous step).
++
+The following configuration in JSON format shows each field and how they are used:
++
+[literal, options="nowrap" subs="+attributes"]
+----
+
+{
+  "file": {
```

Comment:
@jself-sudoku file and env are not supported yet.

Reply:
Thanks. Removed file and env.